tests/suites/test_suite_psa_crypto_metadata.function

```diff
@@ -619,7 +619,7 @@ void key_type( int type_arg, int classification_flags )
 /* END_CASE */
 
 /* BEGIN_CASE */
-void block_cipher_key_type( int type_arg, int block_size_arg )
+void block_cipher_key_type( int type_arg, int block_size_arg, int expecting_power_2 )
```

Comment:
The extra parameter is not useful. All block ciphers must have a block size that's a power of 2.

Reply:
The parameter was for a negative test, so I wrote a test that fails on purpose, and if it successfully detects it is not a power of 2, then it fails. You suggested also that I should remove the .data tests I added. Is it enough then just to have the test but not any special input data?

Reply:
That was a negative test of the test code. In general, we don't need tests of the test code.